lasy/laser.py

```diff
@@ -153,17 +150,17 @@ def propagate(self, distance, nr_boundary=None, backend="NP"):
                     self.prop.append(
                         PropagatorResampling(
                             *spatial_axes,
-                            self.field.omega / scc.c,
+                            omega / scc.c,
                             mode=m,
                             backend=backend,
                             verbose=False,
                         )
                     )
             # Propagate the spectral image
             for i_m in range(self.box.azimuthal_modes.size):
-                transform_data = np.transpose(self.field.field_fft[i_m]).copy()
+                transform_data = np.transpose(field_fft[i_m]).copy()
                 self.prop[i_m].step(transform_data, distance, overwrite=True)
-                self.field.field_fft[i_m, :, :] = np.transpose(transform_data).copy()
+                field_fft[i_m, :, :] = np.transpose(transform_data).copy()
```

Comment:
maybe we could generalize a bit more using  `np.swapaxes(.., time_axis_indx, 0)` instead of `np.transpose`

Reply:
Thanks for the suggestion.
Is it okay if I do it in a separate PR, to keep things clean? 
This is because the current PR focuses on removing the attributes `field_fft`,`omega`, etc.

Reply:
I agree, let's keep it separate.